TEMPLATE/config.py

```diff
@@ -47,6 +51,9 @@ class Config:
     JWT_ACCESS_TOKEN_EXPIRES = timedelta(hours=8)
     CAN_SEED_DB = True
 
+    NPLUSONE_LOGGER = logging.getLogger("app.nplusone")
```

Comment:
is this enabled by default? (I'm 99% sure that observing these extra queries is adding some overhead)

Reply:
how much overhead do you imagine?

Reply:
not much, but would like us to be aware of how much extra (not needed) stuff is done. Also, we are not going to read the logs every day so it's wasted IMHO.

Reply:
IMO if warnings or errors are popping up we should be reading/doing something about them.
it's my belief that some simple accounting is vastly insignificant compared to say, network requests or queries or disk operations. you're right that we shouldn't pile up unneeded stuff. i just think the potential benefit of premature CPU optimization for accounting is outweighed by the potential for performance improvements finding n+1 queries, even or _especially_ on production. they shouldn't be ignored either, and if it turns out they are giving us false positives or things we can't do anything about we can disable it.